src/extension/iSamples_hierarchyFacet.js

```diff
@@ -55,17 +58,17 @@ class HierarchyFacet extends React.Component{
     }
 
     /**
-     * Static json for now.
-     * Will use REST api to get json file from server
-     */
+        * Static json for now.
+        * Will use REST api to get json file from server
+    */
     hierarchy = (label) => {
         switch (label) {
         case "Material":
-            return window.config.vocabularyMaterialType;
+            return material;
         case "Context":
-            return window.config.vocabularySampledFeatureType;
+            return sampledFeature;
         case "Specimen":
-            return window.config.vocabularyMaterialSampleType;
+            return specimanType;
```

Comment:
Let's revert this change.  I can push up to the server now or we can push it all at once if you prefer.

Reply:
Yes I can revert that -- let me know when you push the updated vocab to the server when you are ready.